doc/releases/migration-guide-4.1.rst

```diff
@@ -408,6 +408,12 @@ Bluetooth Host
   makes a device vulnerable for downgrade attacks. If an application still needs to use LE legacy
   pairing, it should disable :kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY` manually.
 
+* The ``CONFIG_BT_SEND_ECC_EMULATION`` Kconfig option has been removed, and the host will now
```

Comment:
Do we need a note for this? The config was introduced in https://github.com/zephyrproject-rtos/zephyr/commit/f4592382bbbc36fdda1a7e60c5d6c8d627840751 which was not in 4.0 as far as I can see.

Reply:
> Do we need a note for this? The config was introduced in [f459238](https://github.com/zephyrproject-rtos/zephyr/commit/f4592382bbbc36fdda1a7e60c5d6c8d627840751) which was not in 4.0 as far as I can see.

Right. I completely missed that point. Then we should be able to remove it directly.